modules/drivers/google/project.clj

```diff
@@ -0,0 +1,28 @@
+(defproject metabase/google-driver "1.0.0-SNAPSHOT-1.30.7"
```

Comment:
I'm surprised to see a `project.clj` file here. Does that play nice with the new `prep` steps?

Reply:
Yeah this was unintentional.  This PR has drug on so long that the `deps.edn` switchover happened in the middle of it, and somehow this file found its way back in.  Probably a borked merge conflict resolution.  Removed now.